### DISCUSSION TITLE
用了一个gpt4.0的key，显示报错为：无效的 API 密钥 提供的 API 密钥不正确

### DISCUSSION THREAD

**Discussion by suiyu4073** [ddiu8081/chatgpt-demo#453] [Q&A]
用了一个gpt4.0的key，显示报错为：无效的 API 密钥 提供的 API 密钥不正确

**Comment by wzmtgggg** [comment]
> 用了一个gpt4.0的key，显示报错为：无效的 API 密钥 提供的 API 密钥不正确

对啊这是为什么😢
